internal/libs/testhelpers/ocr3runner.go

```diff
@@ -170,7 +171,15 @@ func (r *OCR3Runner[RI]) selectLeader() ocr3types.ReportingPlugin[RI] {
 	if numNodes == 0 {
 		return nil
 	}
-	return r.nodes[rand.Intn(numNodes)]
+
+	idx, err := rand.Int(rand.Reader, big.NewInt(int64(numNodes)))
+	if err != nil {
+		panic(err)
+	}
+	if !idx.IsInt64() {
```

Comment:
Is this really needed after using `big.NewInt(int64(numNodes))` above? This is something for a test maybe?

Reply:
it's not really needed, i just added it as a defensive check, not a big deal imo